eslint.js

```diff
@@ -132,6 +132,7 @@ export const config = [
 					parser: (await import('typescript-eslint')).parser,
 					parserOptions: {
 						projectService: true,
+						tsconfigRootDir: import.meta.dirname,
```

Comment:
Wouldn't this resolve to `./node_modules/@epic-web/config`? I don't think that's where we want to configure this root dir to 🤔

Reply:
Ahh okay, I was thinking it would behave differently and resolve to the directory of my eslint.config.js since the epicweb config is spread into our flat eslint config.

After testing and editing the project's epicweb config node module eslint config file, the vscode-eslint LSP used in Helix is still happy when given `/Users/[username]/[[path_to_project]/[project]/node_modules/@epic-web/config` as the tsconfigDir value for some reason.

Anyways, I'll go ahead and close this PR. Sorry for wasting your time!

Reply:
No worries :)